src/config/options.js

```diff
@@ -48,29 +44,17 @@ export const options = {
       ARRAY_EXPRESS: 'ArrayExpress'
     }
   },
-  // setting for the request data form
-  requestDataForm: {
-    radioPediatricCancer: [
-      { label: 'Yes', value: 'Yes' },
-      { label: 'No', value: 'No' }
-    ],
-    radioPrimaryApproach: [
-      { label: 'Bench Research', value: 'Bench Research' },
-      { label: 'Computational Research', value: 'Computational Research' },
-      { label: 'Clinical Research', value: 'Clinical Research' },
-      { label: 'AI/ML Research', value: 'AI/ML Research' }
-    ]
-  },
   // setting for the samples table
   samplesTable: {
     // default values for common queries
     commonQueries: {
       offset: 0,
       limit: 10
     },
-    page: 1,
-    pageSizes: [10, 20, 50]
+    page: 1
   },
+  // setting for the page size
+  pageSizes: [10, 20, 50],
```

Comment:
Although this definition of the page size dropdown was identical, it was defined seperately in `search` and `samplesTable`, so I removed the duplicates.